docs/reference/requirement-sets/powerpoint-api-requirement-sets.md

```diff
@@ -14,8 +14,8 @@ The following table lists the PowerPoint requirement sets, the Office client app
 
 |  Requirement set  |  Office on Windows<br>(connected to a Microsoft 365 subscription)  |  Office on iPad<br>(connected to a Microsoft 365 subscription)  |  Office on Mac<br>(connected to a Microsoft 365 subscription)  | Office on the web |
 |:-----|-----|:-----|:-----|:-----|:-----|
-| [Preview](powerpoint-preview-apis.md)  | Please use the latest Office version to try preview APIs (you may need to join the [Office Insider program](https://insider.office.com)). |
-| PowerPointApi 1.1 | Version 1810 (Build 11001.20074) or later | 2.17 or later | 16.19 or later | October 2018 |
+| [PowerPointApi 1.2](powerpoint-api-1-2-requirement-set.md)  | Version 2011 (Build 13426.20184) or later| not yet<br>supported | 16.43 or later | October, 2020 |
```

Comment:
Onur says its been in "on the web" since October.

Reply:
That's fair, though it's been in "preview" during that time, so it wasn't really available. Of course, this column is just for historical record since you can't be on an old version of PowerPoint for the web anyway...